### PR TITLE
Update to version 0.8.1

### PR DESCRIPTION
**Pull request by rth**
Closes https://github.com/conda-forge/eli5-feedstock/pull/5

### CODE REVIEW

**Comment by rth** [comment]
@conda-forge-admin, please rerender

**Comment by rth** [comment]
Thanks @rmax .

Circle CI is hanging on master.

@conda-forge-admin, please update circle

**Comment by rth** [comment]
Would someone be able to re-start the latest Circle CI job on master please?

**Comment by rmax** [comment]
@rth Thank you for taking care of this. I have cancelled the current job and restarted it.
